AsyncSwift/Identifiable/Schedule.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Schedule.swift
+//  AsyncSwift
+//
+//  Created by Kim Insub on 2022/09/06.
+//
+
+import Foundation
+
+struct Schedule: Identifiable, Codable {
+    var id: Int
```

Comment:
```suggestion
    let id: Int
```

id는 바뀌지 않을테니 let이 적절해보여요

Reply:
적용하도록 하겠습니다 :)